documentation/src/main/doc/modules/connectors/partials/META-INF/connector/smallrye-kafka-incoming.adoc

```diff
@@ -17,6 +17,10 @@ Type: _string_ | false |
 
 Type: _boolean_ | false | `true`
 
+| *tracing-enabled* | Whether tracing is enabled (default) or disabled
+
+Type: _boolean_ | false | `true`
```

Comment:
In Quarkus, it might be a bit more complicated than just `true`. Does it require the tracing extension to be there?

Reply:
No, without an implementation of the API present, it's basically a No-Op

Reply:
Perfect!